etc/rucio.cfg

```diff
@@ -157,33 +158,12 @@ email_from = Rucio <rucio-dev@cern.ch>
 email_test = spamspamspam@cern.ch
 services_list = kafka
 
-[c3po]
-placement_algorithm = t2_free_space
-elastic_url = http://elastic:9200
-redis_host = rucio
-redis_port = 6379
-
-[c3po-popularity]
-elastic_url = http://elastic:9200
-
-[c3po-site-mapper]
-panda_url = http://agis:80/request/pandaqueue/query/list/?json
-ddm_url = http://agis:80/request/ddmendpoint/query/list/?json
-
-[c3po-workload]
-panda_url = http://bigpanda:80/jobs/?category=analysis&jobstatus=running
-window = 604800
-
 [policy]
 package = rubin
-#permission = atlas
+lfn2pfn_algorithm_default = identity
+permission = rubin
```

Comment:
Does this addition 'permission = rubin'  relate to a correponding change in our code base, i.e., the policy that Brandon is maintaining ?

Reply:
I believe `permission` has been added as a requirement in the last couple of releases.  The `etc/rucio.cfg` file here is only used as part of my internal testing.  The one we use for deployment is different.